pandas/tests/window/conftest.py

```diff
@@ -259,3 +261,62 @@ def frame():
         index=bdate_range(datetime(2009, 1, 1), periods=100),
         columns=np.arange(10),
     )
+
+
+def _named_func(name_and_func):
+    name, func = name_and_func
+    if not hasattr(func, "func"):
+        func = partial(func)
+    func.__name__ = name
+    return func
+
+
+@pytest.fixture(
+    params=[
+        _named_func(x)
```

Comment:
i would just write these 2 out as partials as you don't seem to be using this elsewhere

Reply:
Not sure what you have in mind. My aim was to get a properly named (with `__name__`) parameter per window aggregation invocation, so that each invocation appears as a separate test parametrization with a recognizable name. Using just `partial` doesn't give a nicely named result.

Reply:
I find this fixture kinda hard to grock.

Maybe instead you could you use the pattern:

```
# func, kwargs
params = [
    [window_aggregations.roll_sum, {}],
    [window_aggregations.roll_var, {"ddof": 1}]
]
```

and then use the `ids` argument in fixture to give each param item a readable name (I think `ids` may take a callable too)

Reply:
Also I would move this fixture to the new `test_cython_aggregations.py` since this will only be used there

Reply:
I moved the tests and fixture to `test_cython_aggregations.py`. I also refactored the fixture code and added comments, hopefully making it clearer.

Reply:
Incidentally, the refactoring resolved the code-check errors.